### ISSUE TITLE
bot following mouse x,y idea

### ISSUE THREAD

**Issue by Wartoz** [dahquan/rattlesnake#62]
possible?

**Comment by Wartoz** [comment]
we can control the speed of bot and maybe give a direction to bots so they can eat around us or do some protection

**Comment by kurama129** [comment]
i did it already 😀

**Comment by kurama129** [comment]
it collides eat pellet and can boost and also move to mouse

**Comment by kurama129** [comment]
but wartoz why you never share anything and ask everthing

**Comment by David1ali12** [comment]
@kurama129  why dont you pull request it?

**Comment by kurama129** [comment]
ciz yutubers will steal

**Comment by CupidStunts** [comment]
@kurama129 im not seeing you share anything and you just said you have it & the eat food code?
so what makes you any better! people who know me here & from ogar feeder bot.. know that i will share anything. also all i see you do is come here & spam crap & talk down to people like your special. at least wartoz issues are bringing up a topic & hopefully those who are part of the community will share. tbh if your not going to contribute on anything why bother coming here & making the community more negative towards sharing? SHARING IS CARING BRUH! your worst than the DMCA lol.

**Comment by ItsVoxility** [comment]
@cupidstuns Facts bro facts

**Comment by CupidStunts** [comment]
@ItsVoxility he probably lying anyway! but these dudes do my head in!

this should be an opensource friendly community. but it always turns into a competition or someone trying to monetize from it in some way.

anyway i said my piece & well people like him ruin the community!

**Comment by easyrider81** [comment]
I agree. Kudos to those who do share and put work in to make bots open source and help people with problems @dahquan @l3mpik @CupidStunts @mayed505 @anonymousxxx02

**Comment by mayed505** [comment]
Thanks @easyrider81 i actually removed my repo since it youtubers have been stealing it so im waiting for collide to release :+1:

**Comment by kurama129** [comment]
but i know wartoz and he sells i dont need to prove anything and btw if anyone is having a error then im telling a fix github is to design yourself and btw wartoz shares nothing and steals everthing

**Comment by kurama129** [comment]
i shared three time to some users like happy etc they stolenmy agarbot code and cookies and blocked me so i learned never to share personal files with anyone

**Comment by CupidStunts** [comment]
@kurama129 how can you steal open source code? if you gave it to someone without T.O.S or if you actually read the License. people can sell it, if they choose to. not that i would ever, but github is here for people to collaborate or improve the code & find bugs etc.

@easyrider81 kudos for calling out the real sharers on here. there was a couple others to like @ravyn @yario @MrSonicMaster

**Comment by kurama129** [comment]
lol he stole my cookies fb agar bot

**Comment by kurama129** [comment]
and i dont wanna fight

**Comment by mayed505** [comment]
@kurama129 i totally agree. wartoz just want to sell bots i cant believe i helped him with some shitty useless to me code.... anyways hes like selling bots to some kids for $8 or $4 cant remember to who doesnt know its on github.

**Comment by Numan1982** [comment]
wartoz ur a dip shit mate why are u selling ur bots which is an open source?

**Comment by Numan1982** [comment]
u can even code mate so dont think ur all hard and good at coding.

**Comment by CupidStunts** [comment]
yes guys! selling them is bad in my view! but can we stop all the negativity & just keep it friendly.
this repo isnt that old & its already turning into a flame fest of shit!

you never know wartoz might wake up & share something one day lol or maybe you will?
so guys keep it civil plz & dont ruin the community if you can! @youall ;)

i agree with you on selling them, all im saying is though if you want to Bash, Bash a terminal lol

**Comment by mayed505** [comment]
@Numan1982 wartoz is a scammer he scammed one of my friends $4 for proxies now i told him how to refund :D

**Comment by mayed505** [comment]
@CupidStunts i agree on everything but except releasing his files He will neve ever release his files since hes being a cunt.

**Comment by ErmiyaEskandary** [comment]
@kurama129 Please make your code opensource, as required by the license in my bot project as apparently you have made them collide and eat food : https://github.com/ErmiyaEskandary/Slither.io-bot

Sharing is caring ;) That's how the world goes around :+1:

**Comment by mayed505** [comment]
@ErmiyaEskandary i dont think he should do it since youtub skids would steal it and act as their own ones.

**Comment by easyrider81** [comment]
@mayed505 yes I noticed your repo had gone. Shame. Used your bots code, it's good.

@CupidStunts yes I don't know all the good people on here who share and care but they are out there some where. I'm no coder or expert but if I can help someone I will.

**Comment by anonymousxxx02** [comment]
@easyrider81 do you still have mayed bots code ...if yes can you send it to me i want to try his bots

**Comment by easyrider81** [comment]
sure @anonymousxxx02 what's your skype?

**Comment by anonymousxxx02** [comment]
@easyrider81 i dont use skype but you can mail me if you want.... anonymousx023x@gmail.com

**Comment by mayed505** [comment]
@anonymousxxx02 you could have asked me i would had to send it for you.

**Comment by anonymousxxx02** [comment]
@mayed505 sorry bro i thought you didnt share your code with anyone :( .anyway can you send me a copy of your code code in my gmail please???

**Comment by kurama129** [comment]
mayed come in skype

**Comment by anonymousxxx02** [comment]
@mayed505 are you gonna give me a copy of your bots or not ???

**Comment by mayed505** [comment]
@anonymousxxx02 im sorry i cant i really heavily modified it to where i cant release it to anyone

**Comment by easyrider81** [comment]
@anonymousxxx02 I sent you the files I have, sorry for the delay i've been out.
